src/partials/search.html

```diff
@@ -27,7 +27,7 @@
   <label class="md-search__overlay" for="__search"></label>
   <div class="md-search__inner" role="search">
     <form class="md-search__form" name="search">
-      <input type="text" class="md-search__input" aria-label="search" name="query"
+      <input type="text" class="md-search__input" aria-label="{{ lang.t('search.placeholder') }}" name="query"
```

Comment:
Why do we need an `aria-label` when there's already a placeholder set?

Reply:
Ah, sorry, haven't seen that it was already there. Somebody added it at some point. Can we remove this?

Reply:
I don't think we need it, I was just using the translation here (this `aria-label` was there before I came). 😄 

I'll remove that one.